utils/meta.go

```diff
@@ -25,9 +25,6 @@ func NewValue(t reflect.Type) (v reflect.Value) {
 func ToArray(value interface{}) (values []string) {
 	switch value := value.(type) {
 	case []string:
-		if len(value) == 1 && value[0] == "" {
```

Comment:
I tried a test on Go Playground (https://play.golang.org/p/jEOT60xVC-) by commenting these lines and found that they are redundant. Please correct me if I'm wrong.